src/test/resources/archunit.properties

```diff
@@ -0,0 +1 @@
+freeze.store.default.allowStoreCreation=true
```

Comment:
what it will be if we avoid this file creation ?

Reply:
To use the  _freezing rule_  (suppressions) we have to specify this option.

Reply:
this looks like suppress all violations, and there is not itemized suppression. It is not ok.
We will never have time to resolve all at ones.
Please investigate how to have text file, or exclude in imports of classes to resolve to let us resolve problems one by one.

Reply:
@romani I didn't exactly understand. Through this, we can achieve the task that all existing violations are suppressed and only a new violation will be reported.

Reply:
please explain how it become possible if you do not define any of them in config. Please remove your `target` folder and most likely all violations appear again.
If no violations, it means that all violations are suppressed, no matter old or new.

We need a way to clearly see list of current problems in config and have ability to resolve them one by one. Reminder as we do this for pitest, checkstyle violation, pmd, all other tools.

Reply:
Okay, got it. I will look into it.

Reply:
@romani I have implemented a workaround with `java-diff-utils`. Please have a look and let me know if it's okay.